kong/dao/schemas/apis.lua

```diff
@@ -16,11 +17,38 @@ local function validate_target_url(value)
   return false, "Invalid target URL"
 end
 
+local function check_public_dns_and_path(value, api_t)
+  local public_dns = type(api_t.public_dns) == "string" and stringy.strip(api_t.public_dns) or ""
+  local path = type(api_t.path) == "string" and stringy.strip(api_t.path) or ""
```

Comment:
Should this code try to instantiate the `path` parameter as a regular expression to make sure it is valid? Be best to be notified of that when adding a new API as opposed to attempting to consume it.